.github/workflows/publish_sdk.yml

```diff
@@ -33,7 +33,9 @@ jobs:
           virtualenvs-in-project: true
 
       - name: Install dependencies
-        run: poetry install --no-interaction
+        run: |
+          exit 1
```

Comment:
Ooh. What does the exti 1 do?

Reply:
sorry just removed it. Was part of my testing!